src/apps/Altinn.Register/test/Altinn.Register.Tests/Testdata/Ccr/Xml/TestScenarioOverview.md

```
@@ -0,0 +1,1216 @@
+| `TFON` | `WriteTelefon` | S18, S24 |
+| `TFAX` | `WriteTelefax` | ❌ **uncovered** |
+| `FMVA` | `WriteFmva` | S19 |
+| `FADR` | `WriteAddress` | S7, S8, S11, S13, S22, S23 |
+| `PADR` | `WriteAddress` | S13, S22, S23 |
+| `NAVN` | `WriteName` | S10, S11, S22, S23, S24 |
+| `naeringskode` (NACE / SN25) | `WriteNaeringskode` | S5, S22 |
+| `paategning` (PAAT) | `WritePaategning` | S24 (empty form only — content sub-case still uncovered) |
+| `underlagtHjemlandetsLovgivning` (ULOV) | `WriteUlov` | S23 |
+| `registrertHjemlandetsRegister` (UREG) | `WriteUreg` | S20, S23 |
+| `EDAT` / `BDAT` / `NDAT` (date-only) | `WriteSimpleInfoType` (with empty-skip) | S10 (EDAT) |
+| `KATG`, `TKN ` | parser-ignored | n/a |
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)